Branch-SDK/src/io/branch/referral/Defines.java

```diff
@@ -65,7 +64,23 @@
         IsFirstSession("+is_first_session"),
         AndroidDeepLinkPath("$android_deeplink_path"),
         DeepLinkPath("$deeplink_path"),
-        AndroidAppLinkURL("android_app_link_url");
+
+        AndroidAppLinkURL("android_app_link_url"),
+
+        ContentScreen("content_screen"),
+        ContentPath("content_path"),
+        ContentID("$content_id"),
+        CanonicalIdentifier("$canonical_identifier"),
+        ContentTitle("$og_title"),
+        ContentDesc("$og_description"),
+        ContentImgUrl("$og_image_url"),
+        ContentType("$content_type"),
+        PublicallyIndexable("$publicly_indexable"),
+        ContentKeyWords("$keywords"),
+        ContentExpiryTime("$exp_date"),
+        EventTime("event_time"),
+        Params("params"),
```

Comment:
For `v1/register-preview` the payload has `params` as the key for JsonObject representing the UniversalObject properties